lib/open_project/omni_auth/authorization.rb

```diff
@@ -107,9 +107,13 @@ def self.authorize_user_for_provider(provider, &block)
       #                                  including user info and credentials.
       # @yieldparam context The context from which the callback is called, e.g. a Controller.
       def self.after_login(&)
+        ActiveSupport::Deprecation.warn(
+          "after_login does not return the actually logged in user and has been deprecated. " \
+          "Please use omniauth_user_authorized or user_logged_in hooks instead",
```

Comment:
A reference to `OpenProject::Hook` would be useful here too lest someone thinks we are still talking about OmniAuth::Authorization callbacks.

Reply:
Done!